app/scss/questionAdmin.scss

```diff
@@ -1,3 +1,54 @@
 * {
   box-sizing: border-box;
-}
\ No newline at end of file
+}
+
+.container {
+
+  .container_input {
+    float: left;
+    margin: 20px;
+
+    .display-flex {
+      display: flex;
+      align-items: center;
+
+      input {
+        margin-right: 1rem;
+
+        &.primary-button {
+          font-size: 1.4rem;
+          color: $main-background-color;
```

Comment:
Need to add `@import "imports/variables";` at top of file for variables to work

Reply:
Sorted. And box-sizing global also gone